twitter/twitter_types.bal

```diff
@@ -201,6 +202,13 @@ public type Trend record {
     int tweetVolume;
 };
 
+documentation {Struct to define the search request type
```

Comment:
Shall we re-phrase this description? The term struct should be replaced with record. Also reformat the doc comment to start the description in a new line